platformsh-flex-env.php

```diff
@@ -47,6 +47,7 @@ function mapPlatformShEnvironment() : void
     // Map services as feasible.
     mapPlatformShDatabase('database', $config);
     mapPlatformShMongoDatabase('mongodatabase', $config);
+    mapPlatformShRabbitMq('rabbitmq', $config);
```

Comment:
We *just* (like, on Monday) decided on an internal convention for relationship names. :-)  Can you rename this to `rabbitmqqueue`?

Reply:
sure, done.